### PR TITLE
remove all references to atom or atom community

### PR DESCRIPTION
**Pull request by stech11845**
In README at least.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,19 +41,19 @@
 
 [Guidelines]: https://github.com/logos 'Branding Guidelines'
 [Electron]: https://github.com/electron/electron
-[Discord]: https://discord.gg/2tD9evh8qP 'Join the Atom Community Discord today!'
+[Discord]: https://discord.gg/2tD9evh8qP 'Join the Pulsar Discord today!'
```
Comment (confused-Techie) [inline README.md:44]: May be a good idea here to add the proper link to the Pulsar Edit Discord server `https://discord.gg/7aEbB9dGRT`

**Review by confused-Techie — CHANGES_REQUESTED** [review]
This looks great, I've added one change I think we should fix while here, but otherwise appreciate the contribution!

File: README.md (hunk 1 above)
Comment (autumnblazey) [reply README.md:44]: oh yea, that current invite is to atom community's server

Comment (stech11845) [reply README.md:44]: > oh yea, that current invite is to atom community's server

I guess someone should make a Discord, or else we will point people to Atom Community's server.

**Review by confused-Techie — APPROVED** [review]
Thanks for adding the changes! But this looks great to me, while its on Pulsar, since its the readme I'll ping our @pulsar-edit/documentation team to get some additional feedback on what they think. Super rad work!

**Review by autumnblazey — COMMENTED** [review]
hmm, the logo guidelines will need to be updated/removed (since we are not GitHub), and CI will need to be updated once we have working CI again (as per #12), but otherwise I think it looks good!

**Review by autumnblazey — APPROVED** [review]

**Comment by stech11845** [comment]
> he logo guidelines will need to be updated/removed (since we are not GitHub)

**requires:** a new logo.

**Comment by autumnblazey** [comment]
hehe, I think progress is actually being made on that c:

**Comment by confused-Techie** [comment]
So how do we feel about merging this one as is, then once builds are sorted we could modify as needed, and hopefully once the artist being worked with is sorted (or alternative method) we could update that. Since really I'm happy with incremental changes even if we miss a couple things, as long as we know about them

**Comment by stech11845** [comment]
> So how do we feel about merging this one as is, then once builds are sorted we could modify as needed, and hopefully once the artist being worked with is sorted (or alternative method) we could update that. Since really I'm happy with incremental changes even if we miss a couple things, as long as we know about them

Ok.

**Pull request merged**